packages/web/src/components/spa/JobsSection/JobsCard.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export const JobsCard = (props) => {
+    return (
+        <div className="card-content">
```

Comment:
Just add jobs-card div as the outside div here and it should be fine. Please update the css name of this line, it's too generic and people might get conflicting css.